Index: /src/ITGDec/ITGDecod.cpp
===================================================================
--- /src/ITGDec/ITGDecod.cpp	(revision 473)
+++ /src/ITGDec/ITGDecod.cpp	(revision 474)
@@ -534,10 +534,11 @@
 			if (flagoctave) {
 				fprintf(octavefpout, "%7u ", (*infos).seqNum);
 				fprintf(octavefpout, "%u %u %lf ", (*infos).txTime1, (*infos).txTime2, (*infos).txTime3);
 				fprintf(octavefpout, "%u %u %lf ", (*infos).rxTime1, (*infos).rxTime2, (*infos).rxTime3);
 				fprintf(octavefpout, "%u\n", (*infos).size);
+				fflush(octavefpout);
 			}
 
 			//Validate current line
 			errornum = 0;
 			if (((*infos).txTime1 > 24) || ((*infos).txTime1 < 0))
